api_v3/lib/types/KalturaUserEntry.php

```diff
@@ -118,6 +118,16 @@ public function toInsertableObject ( $object_to_fill = null , $props_to_skip = a
 			}
 			$object_to_fill->setKuserId($kuser->getKuserId());
 		}
+
+		$c = new Criteria();
+		$c->add(UserEntryPeer::KUSER_ID, $object_to_fill->getKuserId());
+		$c->add(UserEntryPeer::ENTRY_ID, $this->entryId);
+		$userEntry = UserEntryPeer::doSelect($c);
+		if (count($userEntry) > 0)
```

Comment:
My Tests show that anonymous users can create more than one user entry with this code, I will change it to support types.